plugin/plugins/testPlugin_v2/__init__.py

```
@@ -0,0 +1,1037 @@
+        )
+
+    def _run_on_main_loop_sync(
+        self,
+        coroutine: Any,
+        *,
+        timeout: float,
+        operation: str,
+    ) -> Any | None:
+        loop = self._resolve_main_loop()
+        if loop is None:
+            self.file_logger.warning("{} skipped: main loop is unavailable", operation)
```

Ruff (0.15.5)

[error] Too many arguments for `logging` format string

(PLE1205)